lib/src/html_to_widgets.dart

```diff
@@ -549,9 +554,30 @@ class WidgetsHTMLDecoder {
     final src = element.attributes["src"];
     try {
       if (src != null) {
-        final netImage = await _saveImage(src);
-        return Image(MemoryImage(netImage),
-            alignment: customStyles.imageAlignment);
+        if (src.startsWith("data:image")) {
+          // To handle a case if someone added a space after base64 string
+          List components = src.split("base64, ");
```

Comment:
why split with  base64, ?? you can use simple , to split becasue every base 64 start with that